doc/user_manual/model.tex

```diff
@@ -1345,7 +1345,9 @@ \subsection{EnsembleModel}
         \xmlNode{varName} contains the value of the initial conditions (scalar or arrays, depending of the 
         type of variable). If an array needs to be inputted, the user can specify the attribute  \xmlAttr{repeat} 
         and the code is going to repeat for  \xmlAttr{repeat}-times the value inputted in the body.
-    \end{itemize}
+     \item \xmlNode{initialStartModels}, xmlDesc{XML node, required parameter (if Picard's activated)},
+        specifies the list of models that will be initially executed.
```

Comment:
The manual says that initialStartModels is required, but there does not seem to be a warning or error if it is not present.  (Based on my reading of the code, since ```not []``` is ```True``` it will just use all the vertices as start vertices if not present)

Reply:
Fixed, an error will be raised when the initialStartModels is not provided.